rdt/transformers/categorical.py

```diff
@@ -425,8 +437,41 @@ class LabelEncoder(BaseTransformer):
     values_to_categories = None
     categories_to_values = None
 
-    def __init__(self, add_noise=False):
+    def __init__(self, add_noise=False, order_by=None):
         self.add_noise = add_noise
+        if order_by not in [None, 'alphabetical', 'numerical_value']:
+            raise Error(
+                "order_by must be one of the following values: None, 'numerical_value' or "
+                "'alphabetical'"
+            )
+
+        self.order_by = order_by
+
+    def _order_categories(self, unique_data):
+        if self.order_by == 'alphabetical':
+            if unique_data.dtype.type not in [np.str_, np.object_]:
+                raise Error("The data must be of type string if order_by is 'alphabetical'.")
+
+            nans = pd.isna(unique_data)
+            if nans.any():
```

Comment:
I don't think you need the if statement.

Reply:
If there are no nans then there is no reason to do any of this.